cryptofeed/exchanges/huobi.py

```diff
@@ -32,6 +32,7 @@ class Huobi(Feed):
         L2_BOOK: 'depth.step0',
         TRADES: 'trade.detail',
         CANDLES: 'kline',
+        TICKER: 'ticker'
```

Comment:
the tests fail because this was not imported. Are you sure you tested this code?

Reply:
I tested it in a notebook where I was probably importing the 'TICKER' and 'Ticker' in some other cell, and this hid the issue. My bad, sorry.

I made a minimal code snipped and tested the ticker method again, I hope I haven't missed any issue this time.